Updated various outdated Content Mgmt sections (#1206)
theforeman/foreman-documentation: guides/common/modules/proc_content-view-steps.adoc

> [id="Creating_a_Content_View_{context}"]

Reviewer: Please change the file name to `proc_creating-a-content-view.adoc` (and alway in the assembly).

Author: @maximiliankolb I just realized that I was basically duplicating an already existing section for this. There already is a file name called `proc_creating-a-content-view.adoc`. I went ahead and updated the steps to reflect these changes in that file while also updating the `proc_creating-a-content-view-short.adoc` to get rid of the mention of Component Content View. You will see these changes when I commit again. The assembly has been updated as well with the removal of the file I had initially created.

= Creating a Content View

Use the following procedure to create a Content View.

.Procedure
. Navigate to *Content > Content Views* and click *Create Content View*.

> . Enter a name and click *Create Content View*.

Reviewer: ```suggestion
. Enter a name.
. Optional: Enter a description.
. Click *Create Content View* to create a content view.
```

. On the *Repositories* tab, select a repository from the *Type* list that you want to add to your Content View.
. Optional: Use the *Type* drop-down list to filter for repository types.
. Optional: Use the *Status* drop-down list to filter for added repositories.

> . Select the checkbox next to the available repositories you want to add.

Author: Removing this line.

. Click *Add repositories*.
. Click *Publish new version*.
. Optional: In the *Description* field, enter information about the version to log changes.

> . Optional: You can enable a promotion path by clicking *Promote* to *Select a lifecycle environment from the available promotion paths to promote new version*.

Reviewer: ```suggestion
. Optional: You can enable a promotion path by clicking *Promote* and select a lifecycle environment from the available promotion paths.
```
